### PR TITLE
Update azure-cosmos sql library in flask framework

### PR DESCRIPTION
**Pull request by dgomezc**
## PR checklist

- Quick summary of changes
  - Update azure-cosmos library to 4.2.0
  - Adapt flask framework code to new library changes
- Which issue does this PR relate to?
  - #1224 - Update sql_client.py when Azure SDK for Python v4 be released

### CODE REVIEW

File: templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/backend/sql/sql_client.py [hunk 1]
```diff
@@ -1,44 +1,23 @@
-import azure.cosmos.cosmos_client as cosmos_client
+from azure.cosmos import CosmosClient, errors, http_constants, PartitionKey
 import azure.cosmos.errors as errors
```
Comment (prupipho) [inline templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/backend/sql/sql_client.py]: note: check if it´s needed to import errors and http_constants in both ways/lines. Maybe the imports can be removed?

**Review by prupipho — COMMENTED** [review]
Looks good to me. Will test that later to see if it behaves the same as before.

File: templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/backend/sql/sql_client.py [hunk 2]
```diff
@@ -1,44 +1,23 @@
...
 import azure.cosmos.http_constants as http_constants
```
Comment (prupipho) [inline templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/backend/sql/sql_client.py]:
```suggestion
```

File: templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/backend/sql/sql_client.py (hunk 1 above)
Comment (prupipho) [reply templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/backend/sql/sql_client.py]: Read in [here](http://www.wellho.net/mouth/418_Difference-between-import-and-from-in-Python.html) and I think we can actually get rid of those extra lines

File: templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/backend/sql/sql_client.py (hunk 2 above)
Comment (prupipho) [reply templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/backend/sql/sql_client.py]: in fact... none of those are used anymore? should we still need to add any **try** **except** ?

**Review by prupipho — APPROVED** [review]

File: templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/backend/sql/sql_client.py (hunk 1 above)
Comment (dgomezc) [reply templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/backend/sql/sql_client.py]: This will be resolved in #1618

**Pull request merged**